modal/cli/dict.py

```diff
@@ -118,7 +122,7 @@ async def items(
                 display_item = key, val
             else:
                 cast = repr if use_repr else str
-                display_item = cast(key), cast(val)  # type: ignore  # mypy/issue/12056
+                display_item = _truncate(cast(key)), _truncate(cast(val))  # type: ignore  # mypy/issue/12056
```

Comment:
Could probably combine the `cast` and `_truncate` into a single call but nbd

Reply:
Good point, done!